### DISCUSSION TITLE
No PPC-Mac-OS on UTM (Mac OS)

### DISCUSSION THREAD

**Discussion by horstd** [utmapp/UTM#2428] [UTM Help (macOS)]
Hello, I installed UTM 2.025 on a iMac 2020 (Intel). Want to run a ppc-based Mac-OS, tried it with iso of Mac 9.22 or Mac 10.3 Panther. I always get the error message "no valid state has been set by load or init-program"

The settings are: mac99 based PowerMac, CPU: G4, QENU Machine properties: via=pmu, Boot Order= CD/DVD (also tried hard disk). The iso is in the first position, marked als CD/DVD ISO Image, Interface IDE. I created an new Horst Deinert (disk-0.qcow2).

By starting, the screen shows:
trying hd:,\\:tbxi...
trying hd:,\ppc\bootinfo.txt...
trying Horst Deinert:,,%Boot...
no valid state has been set by load or init-program

In Discussions about Qemu, an argument like "-boot d" should be added, but in the graphic interface of UTM, I find no possibility therefore. 

Can someone help me?

**Comment by brunocastello** [comment]
Go to e-maculation forums for instructions on how to run 9.2.2. And try to use an install disk from Macintosh Garden for 9.2.1 (then upgrade to 9.2.2 later) or just use the macos9lives.com custom install image with a modified ROM.
